manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
So ... I wonder if its time to switch this to true. Right now if you install the module on el7 it fails, because it tries to install 9.2, this flag is actually ignored in a manage_repos = false case btw (different issue, but important to note also).

I feel like this is a major barrier for new users, so maybe worth it?